flow_deps/README.MD

```diff
@@ -0,0 +1,13 @@
+### What is this folder?
+This folder houses the files we pull in from the Estuary/flow project. 
+
+The types should be used along with the server responses.
+
+### How do we update the types?
+For this you'll need `flowctl` installed and the Flow project pulled locally. In the root of the flow project you will need to run the following command:
+`flowctl schemalate typescript --name catalog --hoist-definitions < flow.schema.json > flow.d.ts`
+
+After that you need to copy the `flow.d.ts` file into the `flow_deps` folder.
+
+### How often do we update these files?
+This is not totally decided yet. Probably whenever the folks that "own" this tell us we need to update. This should not happen _all_ the time... but will be frequent enough we'll want to automate this. 
```

Comment:
A couple things to consider here:

> This is not totally decided yet.

The following may be a more concise way of communicating the sentiment: It is still undecided.

> Probably whenever the folks that "own" this tell us we need to update.

The object of this statement is slightly ambiguous. Could you remove the noun this and replace it with the actual descriptor?

> This should not happen all the time... but will be frequent enough we'll want to automate this.

I would recommend replacing the ellipsis with a comma.